packages/mantine/src/components/modal-wizard/ModalWizard.tsx

```diff
@@ -72,25 +72,32 @@ interface ModalWizardProps {
      */
     modalProps?: any;
 
+    /**
+     * Classname for modal
+     */
+    modalClassName?: any;
```

Comment:
Instead of passing the classname as props I think we should leverage Mantine's style system. You can have a read at the documentation section on the subject https://mantine.dev/styles/styles-api/

Reply:
Here's the documentation https://mantine.dev/guides/custom-components/#add-styles-api-support

I think we should stick to `classNames` instead of having a different prop name on every component

Reply:
I followed the documentation and extend on DefaultProps, so I can remove `classNames` prop. I hope it is ok now.